packages/functions/src/oidc/get-vercel-oidc-token.ts

```diff
@@ -23,15 +25,31 @@ import { getContext } from '../get-context';
  * ```
  */
 export async function getVercelOidcToken(): Promise<string> {
+  const env = getEnv(process.env);
   const token =
-    getContext().headers?.['x-vercel-oidc-token'] ??
-    process.env.VERCEL_OIDC_TOKEN;
+    getContext().headers?.['x-vercel-oidc-token'] ?? env.VERCEL_OIDC_TOKEN;
 
   if (!token) {
     throw new Error(
-      `The 'x-vercel-oidc-token' header is missing from the request. Do you have the OIDC option enabled in the Vercel project settings?`
+      `The Vercel OIDC token is missing in your environment. Do you have the OIDC option enabled in the Vercel project settings?`
     );
   }
 
+  if (env.VERCEL_ENV === 'development') {
```

Comment:
consider adding `process.env.NODE_ENV !== "production"` so bundlers will remove that piece of code